apps/asap-server/test/controllers/research-outputs.test.ts

```diff
@@ -609,6 +614,50 @@ describe('ResearchOutputs controller', () => {
       });
     });
   });
+
+  describe('create', () => {
+    afterEach(() => {
+      expect(nock.isDone()).toBe(true);
```

Comment:
is there a reason why you decided to use nock? I think the idea is that we remove nock entirely, or at least for the controller tests, and use the client mock instead for assertions. 

The client could be tested in isolation and that's where we could use nock

Reply:
Looking at the other controller tests, all the REST requests are using nock. I think this will be the first time we will try to use the client mock for REST calls.

Just checking but have we used the client mock for REST calls?

Reply:
ahhh i see, yeah i guess that makes sense, best to use the same method then change - maybe we should create a ticket do address that too? not sure, its not the same as with graphql cause we dont have another way of testing like we do with the schema, but i think there is no value in testing REST at the http level anyway.